Shared/ApiInterface/ApiInterface.cs

```diff
@@ -0,0 +1,53 @@
+﻿using EntityModel;
+using System.Threading.Tasks;
+
+namespace Shared.ApiInterface
+{
+    public interface IApiInterface
```

Comment:
This is the shared interface that is used for API calls and is implemented by CdsInterface and EfInterface